api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -486,22 +490,12 @@ private BadRequestException createBadRequestException(String annotationType, Lon
 
     /**
      * Helper method to create a new {@link CohortReview} and persist it to the workbench database.
-     *
-     * @param workspaceNamespace
-     * @param workspaceId
-     * @param cohortId
+     *  @param cohortId
      * @param cdrVersionId
+     * @param cohort
      */
-    private CohortReview initializeAndSaveCohortReview(String workspaceNamespace,
-                                                       String workspaceId,
-                                                       Long cohortId,
-                                                       Long cdrVersionId) {
-        Cohort cohort = cohortReviewService.findCohort(cohortId);
-        //this validates that the user is in the proper workspace
-        Workspace workspace = cohortReviewService.validateMatchingWorkspace(workspaceNamespace, workspaceId,
-            cohort.getWorkspaceId(), WorkspaceAccessLevel.WRITER);
-        CdrVersionContext.setCdrVersion(workspace.getCdrVersion());
-
+    private CohortReview initializeCohortReview(Long cohortId,
+                                                Long cdrVersionId, Cohort cohort) {
```

Comment:
Maybe throw a newline between these two params for consistency with our other function signatures?

Also - doesn't look like this function needs cohort and cohort Id as separate args, can we grab the ID off the cohort and drop a param?

Reply:
Nice catch. Done.